Lib/gftools/builder/__init__.py

```diff
@@ -71,6 +77,9 @@
     values as demonstrated above, or a dictionary mapping each variable font to a
     per-source list. If neither ``stylespaceFile`` or ``stat`` are provided, a
     STAT table is generated automatically using ``gftools.stat``.
+* ``meta``: A meta table configuration. This should be a dictionary with ``dlng``
+    and/or ``slng`` keys, with the values being a list of script/language tags
+    as defined in the OpenType Specification.
```

Comment:
This might be misleading because they aren’t script tags and language system tags in the usual OpenType sense.